MIT.md

```diff
@@ -0,0 +1,10 @@
+## Copyright 2021, [YOUR NAME]
```

Comment:
Kindly update this to Add your name. It shows that you claim the rights. Replace [YOUR NAME] with your real name.